Unosquare.LedEmotion.Controller/AppClient/Routes/Status.jsx

```diff
@@ -44,9 +58,14 @@ class Status extends Component {
                 <TableCell>Could be wired or wireless</TableCell>
```

Comment:
Missing, use RaspberryIO to check if there's any WIFI Network name related to the current connection. If any use the name, otherwise set Wired as value.